qrl_testing/grpc_compile.py

```diff
@@ -0,0 +1,56 @@
+import grpc
```

Comment:
It does not seem related in any way to the PR or test, is it?

Reply:
We will need this very much for Travis. the docker images clone the QRL main repo, but you don't want the travis test runner to clone the QRL main repo too just for qrl_pb2 and qrl_pb2_grpc, right?